libs/pilight/events/actions/label.lua

```diff
@@ -90,19 +122,35 @@ function M.check(parameters)
 		if nr5 > 0 and nr4 < nr5 then
 			error("label actions are formatted as \"label DEVICE ... TO ... COLOR ... AFTER ...\"");
 		end
+		if nr6 > 0 and nr4 < nr6 then
+			error("label actions are formatted as \"label DEVICE ... TO ... BGCOLOR ... AFTER ...\"");
+		end
+		if nr7 > 0 and nr4 < nr7 then
+			error("label actions are formatted as \"label DEVICE ... TO ... BLINK ... AFTER ...\"");
+		end
 	end
 
 	if nr1 ~= 1 or nr2 ~= 2 then
 		error("label actions are formatted as \"label DEVICE ... TO ...\"");
 	end
 
-	if nr5 > 0 and nr5 ~= 3 then
+	if nr5 > 0 and (nr5 < 3 or nr5 > 5) then
 		error("label actions are formatted as \"label DEVICE ... TO ... COLOR ...\"");
```

Comment:
You said before that you wanted to remove these requirements?

Reply:
Yes indeed, but I assumed that this had to be done in a new PR. 
But ofc. I can do that now.
Shall I remove the order checking completely, or should ``TO`` still be forced to be the first argument in the action?

Reply:
Sorry, I meant to force ``DEVICE`` and ``TO`` to be the first and second argument.

Reply:
Either force everything or nothing.

Reply:
I also propose to make TO optional and check if at least  one of TO, COLOR, BGCOLOR or BLINK are given. That makes it possible to change the visual properties of a label without the need to change the label text.

Reply:
done